apps/bot/src/handleThreadCreation.ts

```diff
@@ -0,0 +1,32 @@
+import { AnyThreadChannel, TextChannel, Client } from 'discord.js';
+import { config } from './config';
+
+const isForumPost = (thread: AnyThreadChannel<boolean>) => {
+  return thread.appliedTags?.length >= 1;
```

Comment:
There's a better way to check is a forum post:

You can check if the `parent` channel of the thread has the type forum

https://discord.js.org/#/docs/discord.js/main/class/ThreadChannel?scrollTo=parent

Reply:
Much better! I'll send a refactor now